src/utils/appData.spec.ts

```diff
@@ -146,3 +146,12 @@ test('Invalid: quote metadata - wrong type', async () => {
 
   expect(validation.result).toBeFalsy()
 })
+
+test('Valid: environment metadata', async () => {
+  const document = {
+    ...BASE_DOCUMENT,
+    metadata: { environment: { version: '0.1.0', name: 'prod' } },
```

Comment:
shouldn't this be `production` as it is in the schema?

> "examples": ["production", "development", "staging", "ens"],

Reply:
That's just an example. The schema is not meant to enforce the contents.